src/abs.cpp

```diff
@@ -278,18 +278,19 @@ List sampler_mc3_abs_tafc_cpp(
 
 //[[Rcpp::export]]
 List Zhu23ABS_cpp(
-    int task_id, // 1 represent estimate task, 2 represent two-alternative choice task
+    int task_id, // 1 represent the fixed stopping rule, 2 represent the relative stopping rule
```

Comment:
Since `switch` in cpp is limited to integers, we will eventually need to transform the stopping rule's name to a number. I would rather keep the current code.

Reply:
didn't know that -- ignore me!